app/src/ui/banners/update-available.tsx

```diff
@@ -44,6 +45,19 @@ export class UpdateAvailable extends React.Component<
   }
 
   private renderMessage = () => {
+    if (this.props.isX64ToARM64ImmediateAutoUpdate) {
+      return (
+        <span onSubmit={this.updateNow}>
+          An optimized version of GitHub Desktop is available for your machine
```

Comment:
Should this reference Apple Silicon/arm64 at all? 

Such as this to be consistent with our marketing page:
Your machine has been detected as an Apple Silicon device. An optimized ...

Reply:
I didn't mention Apple silicon because ideally this would work for Windows too 😆 Do you have a specific suggestion? Knowing that the current text fits very precisely in the minimum window size 😳